script.py

```
@@ -276,12 +333,57 @@ def run_async(coro: Coroutine) -> asyncio.Future | None:
+    Render and return the extension's SSE client JavaScript.
+
+    Loads the ui/ui.js.j2 Jinja2 template and renders it with the configured SSE external path from params.
+
+    Returns:
+        str: Rendered JavaScript; an empty string if the template cannot be loaded or rendered.
+    """
+    import os
+    from jinja2 import Environment, FileSystemLoader
+    js_path = os.path.join(os.path.dirname(__file__), "ui")
+    try:
+        env = Environment(loader=FileSystemLoader(js_path))
```

Ruff (0.15.9)

[error] By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)